### PR TITLE
Further docs changes

### PR DESCRIPTION
**Pull request by mseeger**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/source/tutorials/benchmarking/bm_sagemaker.rst [hunk 1]
```diff
@@ -108,6 +108,9 @@ used with ``hpo_main.py``. For the example above:
 The warm pool feature is most useful with multi-fidelity HPO methods (such as
 ``ASHA`` and ``MOBSTER`` in our example). Some points you should be aware of:
 
+* When using SageMaker managed warm pools with the SageMaker back-end, it is
```
Comment (wistuba) [inline docs/source/tutorials/benchmarking/bm_sagemaker.rst:111]: backend

File: docs/source/examples.rst [hunk 2]
```diff
@@ -179,6 +179,9 @@ You need to `setup SageMaker <faq.html#how-can-i-run-on-aws-and-sagemaker>`__
 before being able to use the SageMaker backend. More details are provided
 in `this tutorial <tutorials/basics/basics_backend.html>`__.
 
+This example can be sped up by using SageMaker managed warm pools, as in
+`this example <#sagemaker-backend-and-checkpointing>`__.
```
Comment (wistuba) [inline docs/source/examples.rst:183]: what's the meaning of double underscore?

Comment (mseeger) [reply docs/source/examples.rst:183]: I use double underscore everywhere. I thought this was the correct way to write things?

File: docs/source/tutorials/benchmarking/bm_sagemaker.rst (hunk 1 above)
Comment (mseeger) [reply docs/source/tutorials/benchmarking/bm_sagemaker.rst:111]: https://en.wikipedia.org/wiki/Frontend_and_backend

Seems like you are right, most of the time.

**Comment by mseeger** [comment]
OK, I did these two changes, which gives a lot of changed files, but it is really just that.

File: docs/source/examples.rst (hunk 2 above)
Comment (wistuba) [reply docs/source/examples.rst]: It apparently works as well

**Review by wistuba — APPROVED** [review]

**Pull request merged**